.github/workflows/editorconfig.yml

```diff
@@ -31,7 +31,7 @@ jobs:
       - name: Setup .NET
         uses: actions/setup-dotnet@v1
         with:
-          dotnet-version: 8.0.x
+          dotnet-version: 10.0.x
```

Comment:
We either need to introduce a variable similar to SOLUTION_NAME. Or add one more workflow file which overrides this (I don't know how include/override is done in github workflows). This file will be overwritten next time we update it in main repo